### ISSUE TITLE
desura: high CPU usage

### ISSUE THREAD

**Issue by karolherbst** [desura/Desurium#241]
the CPU usage of desura is a way to high for me. I'm geting values around 50% every nearly second in htop, but desura is minimized and doing nothing.

For me this is far to high, because it will waste much battery lifetime on laptops.

**Comment by Jookia** [comment]
Maybe it's time we got a profiler?

Also, does this happen in Chromium?

**Comment by karolherbst** [comment]
no, chromium uses around 2% of CPU

**Comment by karolherbst** [comment]
while in offline mode, the CPU usage seems to be normal.

I guess, that this is caused by some HTTPRequest to pull notifications or other things

**Comment by Krod** [comment]
This occurs when changing images of the games featured at the top of the home page client (tab store). On pages with only static content (no gifs, no pictures dynamically changing) the cpu usage remains at about 2%.

This occurs on clients Desura and Desurium. But I noticed that also occurs in browsers Firefox, Chromium and Luakit. Maybe this is not a bug specific Desurium or may not actually be a bug.

To minimize the cpu load check "Unload MainForm on Close" (Performance tab in Settings). Now when you close the client window all the html content is unloaded and the client stays in the system toolbar. But you need to close the client window by close button, click on the icon on the taskbar just hides the client window and not unload the html content. This worked for me.

**Comment by karolherbst** [comment]
Yeah I noticed the same behavior, but in desurium the  load was always higher than with native chromium. But the reason for this could be, that my chromium is either more optimized (compiled with gentoo and CPU optimized options) or because of the newer version of chromium.
